### DISCUSSION TITLE
[Question] How to disable websockets when only deploy a pure frontend app(no backend need)?

### DISCUSSION THREAD

**Discussion by luozhouyang** [reflex-dev/reflex#3409] [Q&A] [answered]
How to disable websockets when only deploy a pure frontend app(no backend need)?

**Comment by ElijahAhianyo** [comment]
Websockets are automatically disabled when you run reflex with `--frontend-only` option

**Reply by luozhouyang** [reply]
But when i export the frontend and deploy it to cloudflare pages, I found this app trying to connect server with websockets
<img width="1110" alt="截屏2024-05-31 20 39 58" src="https://github.com/reflex-dev/reflex/assets/34032031/41014451-be6f-48b4-bb39-fc89cce61ae2">

Here is the website: https://reflex-hello.uagents.app

**Reply by masenf** [reply, marked as answer]
If the app uses any events, then it requires the backend. Even for frontend events like rx.redirect and rx.toggle_color_mode

This is something that we need to fix, but idk when we will be able to get to it.
